plugins/janus_lua.c

```diff
@@ -1346,9 +1353,17 @@ struct janus_plugin_result *janus_lua_handle_message(janus_plugin_session *handl
 	lua_getglobal(t, "handleMessage");
 	lua_pushnumber(t, session->id);
 	lua_pushstring(t, transaction);
-	lua_pushstring(t, message_text);
-	lua_pushstring(t, jsep_text);
-	lua_call(t, 4, 2);
+	uint8_t nargs = 2;
+	if(message_text != NULL) {
```

Comment:
This is going to break plugins if message is NULL but JSEP isn't, because they'll end up reading JSEP where they thing message is. Message must always be passed, even if NULL, because we have a function signature to respect: JSEP can be omitted instead.